### ISSUE TITLE
TooMuchSpaceAfterComma rules corrects my defines

### ISSUE THREAD

**Issue by libreajans** [squizlabs/PHP_CodeSniffer#3531]
I am a newbie in PSR related topics. It may be unrelated PHP_CodeSniffer.

PHP_CodeSniffer aling $variables make them more readable.

For example, this two variable will be converted to the second example. 

```php
$abc = '';
$lognVariable = '':
```

```php
$abc          = '';
$lognVariable = '':
```

On the other hand, I have several defines and I can't aling them. Due to this rules. My alingments will be removed.

```
define('LINK1',        'something');
define('LINK_EXAMPLE', 'something');
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcbf test.php ...`
3. Define alingment will be gone.

**Expected behavior**
I want to **phpcbf** don't touch my aligments if they are define

**Versions (please complete the following information):**
 - OS: Ubuntu 22.04
 - PHP: 8.0
 - PHPCS: 3.6.2 (stable)
 - Standard: PSR-12

**Additional context**
I know I can disable phpcs for my file via //phpcs:disable but I want, at least keep my aligments or phpcs align them.

**Comment by libreajans** [comment]
I decided to follow the TooMuchSpaceAfterComma rule. Thanks.
